_episodes/01-software-development-intro.md

```diff
@@ -80,52 +80,53 @@ request, Controller updates a picture on a user's GitHub profile and then modifi
-> > The above application can be extended by adding new visualisations or new input interfaces 
-> > (by writing a new View or Controller) without having to consider the inner workings of the Model. For example, 
-> > the above DNA program can be extended by adding another Controller 
-> > (a graphical interface which accepts user mouse clicks) and a different View (rendering the visualisation to a displayed window):
-> > ![MVC example of a DNA guide Graphical User Interface application](../fig/mvc-DNA-guide-GUI.png){: width="400px"}
-> > {% comment %}Image from https://www.software.ac.uk/developing-scientific-applications-using-model-view-controller-approach{% endcomment %}
-> > ### Modern Web and Mobile Applications
-> > MVC architecture has become popular for designing web and mobile applications. Users interact with a web/mobile application by sending requests to it via web forms. Requests are processed by the 
-> > Controller, which interacts with the Model to retrieve or update the underlying data. For example, the user may 
-> > request to view its profile/account information or to update their personal details or password. 
-> > Another typical example is adding a 
-> > products into the shopping basket and then proceeding to the checkout to finalise the order. 
-> > Forms to collect users inputs/requests together with 
-> > the info returned and displayed to the user as a result represent the View.  
-> > ### Ordering Food in a Restaurant 
+
+### MVC in Scientific Computing 
+MVC architecture can be [applied in scientific applications](https://www.software.ac.uk/developing-scientific-applications-using-model-view-controller-approach) 
+in the following manner. Model comprises those parts of the application that deal with some type of 
+scientific processing or manipulation of the data, e.g. numerical algorithm, simulation, DNA. View is 
+a visualisation, or format, of the output, e.g. graphical plot, diagram, chart, data table, file.
+Controller is the part that ties the scientific processing and output parts together, mediating input and passing 
+it to the model or view, e.g. command line options, mouse clicks, input files. 
+
+For example, a simple DNA visualisation 
+program has a Controller which accepts command line parameters for the DNA visualisation from users, 
+the Controller then retrieves data from the DNA Model, and uses a single type of View to generate a 
+visualisation - in this case, the view creates a visualisation within a single image file.
+![MVC example of a DNA guide Command Line application](../fig/mvc-DNA-guide-CLI.png){: width="400px"}            
+The above application can be extended by adding new visualisations or new input interfaces 
+(by writing a new View or a new Controller) without having to consider the inner workings of the Model. For example, 
+the above DNA program can be extended by adding another Controller 
+(a graphical interface which accepts user mouse clicks) and a different View 
```

Comment:
This would still be a view in my opinion - a controller should be doing as little as possible, just gluing the views and the models together. Having an MVC architecture with more than one controller is quite unusual.

In this case, I'd see it as the controller launching the GUI view and binding it to a set of models. The GUI would then probably give data directly to its bound models.